rfcs/Py3-first.md

```diff
@@ -0,0 +1,53 @@
+# RFC 65: Switching to "Py3-first" for WPT tests on CI
+
+## Summary
+
+This is to propose switching test suite runs over to Python 3 on the WPT CI (Taskcluster and Azure). At the same time unit tests and infrastructure tests will still be running in Python 2 and Python 3. This is referred to as the “Py3-first" stage in the [Dropping Python 2 support plan](https://github.com/web-platform-tests/rfcs/issues/62).
+
+The minimum python 3 version supported is 3.6 with main focus on 3.8+. For the foreseeable future it is very likely to continue to run unit tests and infrastructure tests on both 3.6 and 3.8+ versions. The WPT test suite itself would run on 3.8+ in the CI.
+
+## Changes on upstream CI
+
+**1. Switch the test suite runs to use Python 3**
+* [Switch wpt run on Taskcluster to Python 3 only](https://github.com/web-platform-tests/wpt/pull/26252) (specifically, Python 3.8)
+
+* Use Python 3.8.x in Azure Pipelines for [Safari](https://github.com/web-platform-tests/wpt/pull/25044) and [Edge](https://github.com/web-platform-tests/wpt/pull/24952) runs.
+
+
+**2. Keep the current CI setup of unit tests and infrastructure tests running on both Py2 and Py3**
+
+The unit tests and infrastructure tests are already running on both Py2 and Py3. Outcomes of unit tests on Py2 and Py3 are not fully identical yet. We have created [a Unit Tests Tracksheet](https://docs.google.com/spreadsheets/d/1__dE5_ABazhDl6ONp76tbkgAimj14gD7QeaUYtfsi_w) to track the differences and give explanations.
+
+
+
+## Recommended actions for downstream (browser vendors)
+
+Switch wpt run to use Python 3, or at least run wptserve with Python 3 as soon as the upstream switches over and stabilizes. This minimizes the risk of exporting new Py2-only handlers to upstream. Waiting until after the upstream completes the switchover reduces churn in case we need to roll back to Py2 temporarily.
+
+## Risks
+
+To assess possible risks and provide evidence for  a move to Python 3 being safe, we have produced [a tracking spreadsheet](https://docs.google.com/spreadsheets/d/1cJcSW8PHe3A4m0dtBfjV4nrYLg67LzIznFbZIknDURc/edit#gid=1054524593). It compares the test results of wpt runs on Python2 and Python 3 for all the supported configurations including:
+* Linux+Chrome Dev
+* Linux+Firefox Nightly
+* Windows + Edge Dev
+* macOS + Safari Technology Preview
+
+Most different test results are caused by the fact that the test itself is flaky. For those flaky tests, links to [flakes](https://wpt.fyi/insights) or results from [recent runs](https://wpt.fyi/runs) are given.
```

Comment:
@ziransun can we update this with recent results from e.g. https://github.com/web-platform-tests/wpt/pull/26252#issuecomment-720607692 ?

That should remove the HTTP/2 support line since that has been added.

Reply:
@stephenmcgruer: Updated the track sheet and removed HTTP/2 statement. Thank you!